navigator/frontend/src/applets/contract/ContractComponent.tsx

```diff
@@ -147,7 +158,11 @@ export default (props: Props): JSX.Element => {
   return (
     <Wrapper>
       <Header>
-        Contract {contract.id} {choicesEl}
+        <LongIdentifier
+          text={`Contract ${shortenContractId(contract.id)}`}
+          identifier={contract.id}
+        />{" "}
```

Comment:
This space here is a bit weird. Isn't it sufficient to replace `{" "}` with just ` `?